journalbeat/magefile.go

```diff
@@ -200,7 +201,12 @@ func installDependencies(pkg, arch string) error {
 		return err
 	}
 
-	return sh.Run("apt-get", "install", "-y", "--no-install-recommends", pkg)
+	if arch == "i386" {
```

Comment:
Why the special case of running a script for i386? Seems like you should be able to run a different command (or with different params) directly from Go.

Reply:
We need to send `Yes, do as I say!` to stdin of apt-get. Otherwise installation fails. Even if `--force-yes` is used.

Reply:
With recent changes to debian 8 packages one can not install `libsystemd-dev:i386` without `libsystem0:i386` anymore :(

It's a workaround for now.

Yet I wonder if that's the case for i386 only or other architectures as well.

Reply:
I'd prefer to not have the external script, but no need to address this with it being a quick fix and hopefully temporary once https://github.com/elastic/golang-crossbuild/issues/23 is addressed.

The script could probably be avoided by either running something like `sh.Run("/bin/sh", "-c", ...)` or by directly using `exec.Command()` and writing the necessary string to stdin.